jlm/util/TarjanScc.hpp

```diff
@@ -89,9 +90,14 @@ FindStronglyConnectedComponents(
             dfsStack.push(next);
         }
       }
-      else // This is the second time node is visited, all children have been processed
+      else if (order[node] == SCC_FINISHED) // This node has already been fully processed
```

Comment:
I like that much better in terms of readability.

Reply:
@phate it looks like this comment might have started a new review that is not accepted? I'm not able to merge it seems